crates/glaredb/src/local.rs

```diff
@@ -226,6 +233,7 @@ impl LocalSession {
                         self.opts.mode,
                         self.opts.max_width,
                         self.opts.max_rows,
+                        now,
```

Comment:
This means only `ExecutionResult::Query` statements will be timed, so for now no DDL/DML timing info.